files/chef-marketplace-ctl-commands/register_node.rb

```diff
@@ -1,6 +1,16 @@
 require 'highline/import'
+require 'tempfile'
 
 add_command_under_category 'register-node', 'Setup', 'Register node with Chef to enable support', 2 do
+  def run_chef_non_root(dna, args = '')
+    run_command('mkdir -p /tmp/chef-marketplace/local-mode-cache/nodes')
+    run_command('rm -rf /tmp/chef-marketplace/local-mode-cache/nodes/*')
```

Comment:
Nitpick, but it always rubs me the wrong way to see us shell out when there are Ruby-equivalents in FileUtils, etc.